docs/examples/general/data_loading/dataloading_webdataset.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<p>Maybe it is boring but creating a copy paste from https://docs.nvidia.com/deeplearning/dali/user-guide/docs/examples/general/data_loading/dataloading_tfrecord.html but with WebDataset reader instead would be more educational.</p><p>The common user would see that he can do the same things with just a small change in the reader to move to different dataset format.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I did that originally but removed it because of the usage of MNIST</p>

Reply:
<p>MNIST is good if you want to train something, but our toy classification dataset is better for visualization.</p>

Reply:
<p><code>In this example, we augment the data by applying noise and blurring.</code>  needs to be updated</p>

Reply:
<p>alright</p>